test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -536,18 +536,20 @@ test.meta('v3', false)
       record => record.response.statusCode === 200 &&
         record.request.url.match(/poll/)
     )).eql(3);
+    logger.clear();
```

Comment:
Why did we need to add these now when before it worked without them?

Reply:
Actually, I am not totally sure lol, what was happening was the logger object contained a bunch of the previous logs which caused the final assertion to have 16+ messages in them. After clearing the logs (after each log assertion), it brought the final assertion down to what was expected which was 5 log messages (the additional one is caused by auth-js). But I am not sure what v2 is doing that made it just completely ignore the others 🤔 its super perplexing 😢

Reply:
so this ticket https://oktainc.atlassian.net/browse/OKTA-587189 will fix that 1 extra poll, but we still need to clear the logger each time?

Reply:
Yep, clearing the logger prevents it from containing log messages from earlier parts of the test.

Reply:
Without it, the last assertion has double the amount of logs in it and has a large variance compared to v2.